charts/corda/values.yaml

```
@@ -91,6 +91,8 @@ db:
           # -- the cluster database password secret key
           key: ""
     database: cordacluster
+    # the schema in which the cluster database will be stored
```

without the `--` this doesn't get included in the generated README (it's usually just on the leaf nodes in the tree)
```suggestion
    # -- the schema in which the cluster database will be stored
```